src/js/view/controls/components/menu/menu.js

```diff
@@ -53,58 +54,49 @@ export default class Menu extends Events {
     }
     createItemsContainer() {
         const itemsContainerElement = this.el.querySelector('.jw-settings-submenu-items');
-        const getTopbar = () => this.topbar;
+        const closeButtonElement = this.mainMenu.closeButton && this.mainMenu.closeButton.element();
+        const backButtonElement = this.mainMenu.backButton && this.mainMenu.backButton.element();
+        const categoryButtonElement = this.categoryButton && this.categoryButton.element();
+        const getTopbar = () => this.topbar && this.topbar.firstChild;
         const onKeydown = (evt) => {
             if (event.target.parentNode !== itemsContainerElement) {
                 return;
             }
-            const focusElement = (ele, i) => {  
-                if (ele) {
-                    ele.focus();
-                } else if (i !== undefined) {
-                    itemsContainerElement.childNodes[i].focus();
-                }
-            };
+
             const { sourceEvent, target } = evt;
-            const isFirstChild = itemsContainerElement.firstChild === target;
-            const isLastChild = itemsContainerElement.lastChild === target;
             const topbar = getTopbar();
-            const nextItem = closeButtonElement || nextSibling(topbarTarget);
-            const prevItem = backButtonElement || previousSibling(topbarTarget);
-            const nextSubItem = nextSibling(evt.target);
-            const prevSubItem = previousSibling(evt.target);
+            const settingsElement = document.getElementsByClassName('jw-icon-settings')[0];
```

Comment:
Will it work if we have 2 players on the site?

Reply:
Not always, but this was ported directly from the original implementation, so any bugs surfaced from this already exist in prod. I agree there should be some form of linking of the toggle button and the main menu, but that's beyond the scope of the refactor.

Reply:
I tested and after closing the menu in #2 player focus is on #1 player.
#2 Player
Open menu > keyDown > ESC 
The problem only occurs when I use the up or down key.